pkgs/applications/misc/pdfarranger/default.nix

```
@@ -4,16 +4,17 @@
 , python3Packages
 , gtk3
 , poppler_gi
+, libhandy
 }:
 
 python3Packages.buildPythonApplication rec {
   pname = "pdfarranger";
   version = "1.10.1";
 
   src = fetchFromGitHub {
-    owner = pname;
-    repo = pname;
-    rev = "refs/tags/${version}";
+    owner = "pdfarranger";
+    repo = "pdfarranger";
+    rev = version;
```

I think we should keep
```suggestion
    rev = "refs/tags/${version}";
```

---

> I think we should keep

Ok I've reverted it back to that.